index.html

```diff
@@ -119,6 +119,33 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
+        
+        <!-- ________ txlan1710 card START ________  -->
+        <div class="card">
+          <p class="name">Ta Xuan Lan</p>
+          <p class="contact">
+            <i class="fab fa-x-twitter"></i>
+            <a href="https://twitter.com/TB2014929" target="_blank">Tb2014929</a>
+            <i class="fab fa-github"></i>
+            <a href="https://github.com/txlan1710" target="_blank">txlan1710</a>
+          </p>
+          <p class="about">about you</p>
```

Comment:
Hi, please also update the description about yourself and push again, then we'll review and merge your PR, thanks
```suggestion
          <p class="about">Any text about yourself</p>
```